media/css/mozorg/home/includes/ctd-promo-de.scss

```diff
@@ -0,0 +1,52 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I don't think this file should be in /includes because it doesn't get imported into another style sheet.